synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -265,6 +271,12 @@ export class SynopsysToolsParameter {
       },
     };
 
+    if (isBoolean(inputs.BLACKDUCK_WAIT_FOR_SCAN)) {
```

Comment:
can we make use of any one of between parseToBoolean and isBoolean in all scans?

or can we simply assign true inside if block?

if (isBoolean(inputs.BLACKDUCK_WAIT_FOR_SCAN)) {
blackduckData.data.blackduck.waitForScan = true
    }

Reply:
I think we can not simply assign true inside if block because user may pass false value too.